src/components/LoginStatus.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import slice, { UserState, LoadingState, User } from "../slices/userSlice";
+import { connect } from "react-redux";
+
+interface OwnProps {

```

Comment:
Naming is a bit generic here. I would suggest to go with something more specific, like `LoginStatusProps`.